### DISCUSSION TITLE
Syncing issues in Windows?

### DISCUSSION THREAD

**Discussion by InternetGuy140** [Chia-Network/chia-blockchain#1697] [Q&A]
edit: this did eventually sync. I'm not sure really what got it going. I had to restart a few times and it finally took. A little afraid when I reinstall this week (rebuilding rig), lol.

I'm having a terrible time in trying to get started running into multiple issues. Mostly plotting, but now having syncing issues. Initially, even after 24hrs, I was not getting very far. Every restart of the GUI would advance a little. And the total block height would keep getting higher, so it was updating there. I turned on port forwarding 8444 to me, and boom, things advanced, I synced in a day or less. Got 3 plots in. This was last wed-sat.

The past couple of days, in trying to fix plotting, I just tried reinstalling chia and clearing my .chia directory (but not enough? It still knew who which key I using). I'm back at the slow climb to sync (I have not re-added my old plots).  At one point, in the logs, I could see a lot of rejected blocks notes and it kept falling back to a previous block and starting all over. Eg, I get to 14000. but eventually, in the logs, I see it request back to 10000 in chunks back up to 14000. And it'd repeat. I have no idea if that's normal? Other stuff was going on back and forth between other nodes, but I did not save the log :(

I blew it up again. And got stuck at like 512. Block height was getting updated still. On a whim, I decided to make one new plot (still haven't re-added the old ones).  The syncing started back up again! All the way to 36032. And now I'm stuck again.
But now it's a lot of
`full_node src.full_node.full_node : ERROR    Error with syncing: <class 'concurrent.futures.process.BrokenProcessPool'>Traceback (most recent call last):
  File "src\full_node\full_node.py", line 620, in _sync
  File "src\full_node\full_node.py", line 661, in sync_from_fork_point
  File "src\full_node\full_node.py", line 721, in receive_block_batch
  File "src\consensus\blockchain.py", line 491, in pre_validate_blocks_multiprocessing
  File "src\consensus\multiprocess_validation.py", line 231, in pre_validate_blocks_multiprocessing
  File "asyncio\base_events.py", line 755, in run_in_executor
  File "concurrent\futures\process.py", line 625, in submit
concurrent.futures.process.BrokenProcessPool: A child process terminated abruptly, the process pool is not usable anymore
`
It's like I get hit with a new thing all the time.  

I've memtest86 over the weekend. Did prime for a few hours. All seem fine.

**Comment by ghost** [comment]


**Reply by dorkmo** [reply]
typically you have to set up port forwarding in your network's router
